blaze/compute/tests/test_pandas_compute.py

```diff
@@ -40,7 +40,7 @@ def df_all(a_df, b_df):
     This method doesn't check the index which can be manipulated during
     operations.
     """
-    assert all(a_df.columns == b_df.columns)
+    assert np.all(a_df.columns == b_df.columns)
```

Comment:
Any reason we shouldn't gut this function and just use `tm.assert_frame_equal(a_df, b_df)`? The use of this later in this file looks redundant.